scripts/util.ts

```diff
@@ -19,13 +20,20 @@ export function transformFile(
 const versionOrder = ['major', 'minor', 'patch'];
 
 export async function nextSdkVersion(): Promise<string> {
-  const currentVersion = currentSdkVersion
-    .split('.')
-    .map(num => parseInt(num, 10));
+  const currentVersion = getPackageVersion();
   const releasePlan = await getReleasePlan(process.cwd());
-  const versionIncrease = releasePlan.releases
+
+  const versionIncreases = releasePlan.releases
     .map(({ type }) => versionOrder.indexOf(type))
     .sort((a, b) => b - a);
-  currentVersion[Math.min(...versionIncrease)] += 1;
-  return currentVersion.join('.');
+  const release = versionOrder[Math.min(...versionIncreases)];
```

Comment:
Kind of unintuitive that the "bigger" release (e.g. major > minor > patch) has the lowest index IMO, but I guess that is [pp].